ffcx/ir/uflacs/elementtables.py

```diff
@@ -571,19 +564,6 @@ def optimize_element_tables(tables,
         unique_tables[uname] = tbl
 
     unique_table_origins = {}
-    # for ui in range(len(unique_tables_list)):
-    for ui in []:  # FIXME
-        uname = unique_names[ui]
-
-        # Track table origins for runtime recomputation in custom integrals:
-        name = "name of 'smallest' element we can use to compute this table"
-        dofrange = "FIXME"  # table_ranges[name]
-        dofmap = "FIXME"  # table_dofmaps[name]
-
-        # FIXME: Make sure the "smallest" element is chosen
-        (element, avg, derivative_counts, fc) = table_origins[name]
-        unique_table_origins[uname] = table_origin_t(element, avg, derivative_counts, fc, dofrange,
```

Comment:
I think table origins can also be removed from the ir as it was only used to work out how to add permutations to preintegrated tables. (Again this is probably to be done in a future PR)